events/tracks/gta2024-bycovid.md

```diff
@@ -42,17 +42,17 @@ program:
       - type: custom
         name: "[An automated SARS-CoV-2 genome surveillance system built around Galaxy](https://www.infectious-diseases-toolkit.org/showcase/covid19-galaxy)"
         description: |
-          Lecture Video
+          [Lecture Tutorial](https://www.infectious-diseases-toolkit.org/showcase/covid19-galaxy)
```

Comment:
is there a video there?

Reply:
Yes, it should be added now! Thanks for checking.